src/main/java/ttt/players/PlayerFactory.java

```diff
@@ -17,20 +14,20 @@
         Player player1;
         Player player2;
         if (type == 1) {
-            player1 = new HumanPlayer(Marks.X);
-            player2 = new HumanPlayer(Marks.O);
+            player1 = new HumanPlayer(Marks.X, new ConsoleInput());
```

Comment:
You could create one instance of ConsoleInput() and use it in each player you create?